doc/changelogs/CHANGELOG_V17.md

```diff
@@ -39,6 +40,141 @@
+* \[[`f1009ae9f9`](https://github.com/nodejs/node/commit/f1009ae9f9)] - **test**: fix typographical error (Rich Trott) [#41983](https://github.com/nodejs/node/pull/41983)
+* \[[`8cf918125c`](https://github.com/nodejs/node/commit/8cf918125c)] - **test**: fix intl tests on small-icu builds (Antoine du Hamel) [#41939](https://github.com/nodejs/node/pull/41939)
+* \[[`f4304ea941`](https://github.com/nodejs/node/commit/f4304ea941)] - **test**: improve code coverage for streams/duplexify (Erick Wendel) [#41862](https://github.com/nodejs/node/pull/41862)
+* \[[`dc14bd85ee`](https://github.com/nodejs/node/commit/dc14bd85ee)] - **test**: remove lint rule for setTimeout() arguments (Rich Trott) [#41901](https://github.com/nodejs/node/pull/41901)
+* \[[`76f0f1c06d`](https://github.com/nodejs/node/commit/76f0f1c06d)] - **test**: move test-crypto-engine to addon (Michael Dawson) [#41830](https://github.com/nodejs/node/pull/41830)
+* \[[`c498a72cff`](https://github.com/nodejs/node/commit/c498a72cff)] - **tls**: avoid throw in onerror for bad TLSSocket obj (Valters Jansons) [#41523](https://github.com/nodejs/node/pull/41523)
+* \[[`48fa1e5671`](https://github.com/nodejs/node/commit/48fa1e5671)] - **tools**: enable es2022 env in ESLint config (Michaël Zasso) [#42043](https://github.com/nodejs/node/pull/42043)
+* \[[`0f4f2ddfc2`](https://github.com/nodejs/node/commit/0f4f2ddfc2)] - **tools**: add `console` to the list of restricted globals (Antoine du Hamel) [#42012](https://github.com/nodejs/node/pull/42012)
+* \[[`dcb51b0a48`](https://github.com/nodejs/node/commit/dcb51b0a48)] - **tools**: lint deprecation codes (Antoine du Hamel) [#41992](https://github.com/nodejs/node/pull/41992)
+* \[[`98167e884b`](https://github.com/nodejs/node/commit/98167e884b)] - **tools**: fix bugs in prefer-primordials linter rule (Antoine du Hamel) [#42010](https://github.com/nodejs/node/pull/42010)
+* \[[`01618c621e`](https://github.com/nodejs/node/commit/01618c621e)] - **tools**: add timers functions to the list of restricted globals (Antoine du Hamel) [#42013](https://github.com/nodejs/node/pull/42013)
+* \[[`297d217efc`](https://github.com/nodejs/node/commit/297d217efc)] - **tools**: reduce list of globals in ESLint config (Rich Trott) [#41996](https://github.com/nodejs/node/pull/41996)
+* \[[`916cda4afc`](https://github.com/nodejs/node/commit/916cda4afc)] - **tools**: update remark-preset-lint-node to 3.3.1 (Rich Trott) [#41999](https://github.com/nodejs/node/pull/41999)
+* \[[`7a72aa776a`](https://github.com/nodejs/node/commit/7a72aa776a)] - **tools**: update ESLint to 8.9.0 (Luigi Pinca) [#41973](https://github.com/nodejs/node/pull/41973)
+* \[[`0c4977cf29`](https://github.com/nodejs/node/commit/0c4977cf29)] - **tools**: update lint-md-dependencies to rollup\@2.67.2 (Node.js GitHub Bot) [#41941](https://github.com/nodejs/node/pull/41941)
```

Comment:
I haven't checked all of them, but it looks like there's a mismatch to the commit shas on the actual branch, most probably due to one of the rebases.

Reply:
Fixed in latest push.